src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -642,6 +643,12 @@ void SettingsLoader::_parse(const OriginTag origin, const winrt::hstring& source
                     // versions of these themes overriding the built-in ones.
                     continue;
                 }
+
+                if (origin != OriginTag::InBox)
+                {
+                    static std::string themesContext{ "themes" };
+                    theme->LogSettingChanges(settings.themesChangeLog, themesContext);

```

Comment:
Should we do the same for color schemes?

Reply:
So, color schemes have the color table, foreground, background, selectionBackground, cursorColor, and name. And they're all stored as `Color`s, not optionals. The only one that I could see being valuable there is `selectionBackground`, but that seems like more work than it's worth, especially considering that we serialize all of them.

I think we can revisit this one later if we start messing with color scheme settings (and as we get closer to auto-save).

@DHowett @nguyen-dows Thoughts?

Reply:
I think we can ignore schemes. We can tell if users are customizing their schemes by how much they change _profile.colorScheme_